src/components/entry/EntryNav.js

```diff
@@ -1,7 +1,70 @@
-const EntryNav = () => (
-  <>
-    <p>EntryNav</p>
-  </>
-)
+import { ArrowBack } from '@styled-icons/boxicons-regular'
+import { Pencil } from '@styled-icons/boxicons-solid'
+import { MapMarkerAlt as Map } from '@styled-icons/fa-solid'
+import styled from 'styled-components'
+
+import { theme } from '../ui/GlobalStyle'
+import IconButton from '../ui/IconButton'
+
+const EntryNavContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const EntryNavTextContainer = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+`
+
+const EntryNavText = styled.p`
+  margin-left: 10px;
+`
+
+const EntryNavIconsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 110px;
+`
+
+const EntryNav = () => {
+  const onEditEntryDetailsButtonClick = () => {
+    // TODO: edit entry callback
+    console.log('Edit entry details clicked')
+  }
+
+  const onViewEntryOnMapButtonClick = () => {
+    // TODO: view entry on map callback
+    console.log('View entry on map clicked')
+  }
+
+  return (
+    <EntryNavContainer>
+      <EntryNavTextContainer>
+        <ArrowBack size="20" color={theme.secondaryText} />
+        <EntryNavText>Results</EntryNavText>
+      </EntryNavTextContainer>
+      <EntryNavIconsContainer>
+        <IconButton
+          size={50}
+          raised={false}
+          icon={<Pencil size="20" color={theme.secondaryText} />}
+          onClick={onEditEntryDetailsButtonClick}
+          label={'edit-entry-details'}
+        />
+        <IconButton
+          size={50}
+          raised={false}
+          icon={<Map size="20" color={theme.secondaryText} />}
```

Comment:
I think it'd be a good idea to style the icon itself (you can select with `svg {` in StyledIconButton) to be height 50% of container. This way we don't have to pass in an explicit size for the icon, just a size in IconButton. You can ref Siraj's comments in the design channel:

> Hey guys — for resizable components (like IconButton) we should have size be a prop that takes a number.  Centered child icons can have a width and height of 50% of the size. Since these components are used frequently at many sizes throughout the interface its beneficial to be able to modify them for each use case.

I believe @SirajChokshi mentioned to only set one of the dimensions of the icon to 50%, just in case the icon is rectangular.

Reply:
whoops I completely forgot about this, thanks!